app/test/unit/main-process/spell-checker-menu-test.ts

```diff
@@ -0,0 +1,58 @@
+import { getSpellCheckLanguageMenuItemOptions } from '../../../src/main-process/menu/build-spell-check-menu'
+import { getAvailableSpellcheckerLanguages } from '../../helpers/menus/available-spellchecker-languages-helper'
+
+describe('spell-checker-menu', () => {
+  it('spell checker language is english && user language is english', () => {

```

Comment:
For the test description, we use sentences that describe how the subject under test should behave (where `it` is the subject under test):
```suggestion
  it('returns null menu item options when both spellchecker language and user language are english', () => {

```